data/world_countries_v7.topo.json

```
@@ -15,6 +15,7 @@
           "properties": {
             "iso2": "AD",
             "iso3": "AND",
+            "iso_numeric": 20,
```

I think these should be strings.

For example
```suggestion
            "iso_numeric": "020",
```